deploy/helm/kubernetes/templates/apiserver-deployment.yaml

```diff
@@ -28,7 +32,7 @@ spec:
         - --etcd-cafile=/pki/apiserver-etcd-client/ca.crt
         - --etcd-certfile=/pki/apiserver-etcd-client/tls.crt
         - --etcd-keyfile=/pki/apiserver-etcd-client/tls.key
-        - "--etcd-servers=https://{{ $fullName }}-etcd-0.{{ $fullName }}-etcd:2379,https://{{ $fullName }}-etcd-1.{{ $fullName }}-etcd:2379,https://{{ $fullName }}-etcd-2.{{ $fullName }}-etcd:2379" # TODO
+        - --etcd-servers={{- range $k, $v := until (atoi (printf "%d" (int64 $.Values.etcd.replicas))) }}https://{{ $fullName }}-etcd-{{ $k }}.{{ $fullName }}-etcd:2379,{{ end }}
```

Comment:
I think better to move this into [named template](https://helm.sh/docs/chart_template_guide/named_templates/) in `_helpers.tpl` and call like that:
```yaml
- --etcd-servers={{ template "kubernetes.etcdServers" . }}
```

Ending comma should be solved

Reply:
Example template:

```yaml
{{- define "kubernetes.etcdServers" -}}
  {{- $local := dict "first" true -}}
  {{- range $k, $v := until (atoi (printf "%d" (int64 $.Values.etcd.replicas))) }}
  {{- if not $local.first -}},{{- end -}}
  https://{{ template "kubernetes.fullname" . }}-etcd-{{ $k }}.{{ template "kubernetes.fullname" . }}-etcd:2379
  {{- $_ := set $local "first" false -}}
{{- end }}
```
(not checked this yet)